docs/SETUP.md

```diff
@@ -77,21 +77,48 @@ The next step is to setup the backend. The setup instructions depends on which b
 
 # Setup Firebase Backend
 TODO
-1. Install Firebase Tools
-2. Get Firebase API Keys
+1. Install packages
+    ```
+    pip3 install firebase_admin
+    ```
+
+2. Generate Firebase API key
+    If there is already an existing API key, please ask whoever owns it to give it to you (not through git). In the future, we will figure out a better way of doing this. Otherwise, you need to generate a new Firebase API key:
+
+    a. Go to the [Firebase console](https://console.firebase.google.com/u/1/project/startercode-98c1c/)
+    b. Click on Settings > Service accounts > Generate new private key
+    c. Share it with everyone that needs the private key.
+
+3. Place in starter-code repository
+
+    Copy the contents into `backend/db/config/serviceAccountCredentials.json`
+
+4. Start the Flask application and make sure the database is working
+    ```
+    $ cd <my-project>
+    $ pipenv shell
+    $ flask run
+    ```
+    In the firebase console, click on Database > Cloud Firestore.
+
 
 # Setup Python-Flask-Postgres Backend
 1. Install packages
     ```
     $ pip3 install flask_sqlalchemy
     ```
 
-2. Create the database
```

Comment:
don't we need to first start the postgres server? Can you add the pg_ctl .... command here?

Reply:
done